src/yea_wandb/plugin.py

```diff
@@ -111,6 +127,15 @@ def parse_term(v, state, result=None):
     return found
 
 
+def contains_regex(iterable, pattern):
```

Comment:
add a trivial test that demonstrates how this is used in:
https://github.com/wandb/yea-wandb/blob/main/tests/plugin_test.py